src/bootstrap/jobs/udp_tracker.rs

```diff
@@ -8,30 +8,34 @@
 //! for the configuration options.
 use std::sync::Arc;
 
-use log::{error, info, warn};
 use tokio::task::JoinHandle;
 use torrust_tracker_configuration::UdpTracker;
 
 use crate::core;
-use crate::servers::udp::server::Udp;
+use crate::servers::udp::server::{Launcher, UdpServer};
 
 /// It starts a new UDP server with the provided configuration.
 ///
 /// It spawns a new asynchronous task for the new UDP server.
+///
+/// # Panics
+///
+/// It will panic if the API binding address is not a valid socket.
+/// It will panic if it is unable to start the UDP service.
+/// It will panic if the task did not finish successfully.
 #[must_use]
-pub fn start_job(config: &UdpTracker, tracker: Arc<core::Tracker>) -> JoinHandle<()> {
-    let bind_addr = config.bind_address.clone();
+pub async fn start_job(config: &UdpTracker, tracker: Arc<core::Tracker>) -> JoinHandle<()> {
+    let bind_to = config
+        .bind_address
+        .parse::<std::net::SocketAddr>()
+        .expect("Tracker API bind_address invalid.");
+
```

Comment:
Hi @da2ce7, this relates to my previous comment. Maybe at this point `UdpTracker` configuration should contain a valid `SocketAddr`.

Reply:
@josecelano fixed.